spec-main/api-safe-storage-spec.ts

```diff
@@ -0,0 +1,113 @@
+import * as cp from 'child_process';
+import * as path from 'path';
+import { safeStorage } from 'electron/main';
+import { expect } from 'chai';
+import { emittedOnce } from './events-helpers';
+import { ifit } from './spec-helpers';
+import * as fs from 'fs';
+
+describe('safeStorage module', () => {
+  after(async () => {
+    if (process.platform === 'linux') return;
+    const pathToEncryptedString = path.resolve(__dirname, 'fixtures', 'api', 'safe-storage', 'encrypted.txt');
+    if (fs.existsSync(pathToEncryptedString)) {
+      await fs.unlinkSync(pathToEncryptedString);
+    }
+  });
+
+  /* isEncryptionAvailable returns false in Linux when running in headless mode (on CI). Headless mode stops
+   * Chrome from reaching the system's keyring or libsecret. When running the tests with config.store
+   * set to basic-text, a nullptr is returned from chromium,  defaulting the available encryption to false.
+   * Thus, we expect false here when the operating system is Linux.
```

Comment:
This isn't related to any "headless" mode rather that we run our tests with a mocked dbus

Reply:
That's my mistake, added that comment early and I didn't change it to reference the dbus mock 🙇‍♀️